lib/AST/RequirementMachine/RequirementMachineRequests.cpp

```diff
@@ -782,7 +782,6 @@ InferredGenericSignatureRequest::evaluate(
 
   unsigned numOuterParams = genericParams.size();
   if (isExtension) {
-    assert(allowInverses);
```

Comment:
Heh, what a pointless assertion

Reply:
Yeah I was thinking of calling it "expandDefaults" since that's the main effect.